v1/python-sdk/tutorials/automl-with-azureml/forecasting-demand-forecasting-tcn/auto-ml-forecasting-demand-tcn.ipynb

```diff
@@ -0,0 +1,1472 @@
+    "To trigger an inference pipeline run, we first need a running environment that contains all the appropriate packages for the model unpickling. This environment can be either assessed from the training run or using the `yml` file that comes with the model. In this section we use the environment from the training run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from scripts.register_model import get_best_automl_run\n",
+    "\n",
+    "best_run = get_best_automl_run(training_pipelin_run)\n",
+    "inference_env = best_run.get_environment()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core.runconfig import RunConfiguration\n",
+    "\n",
+    "run_config = RunConfiguration()\n",
+    "run_config.environment = inference_env"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6.1.2 Build and submit the inference pipeline\n",
+    "\n",
+    "The inference pipeline will create two different output formats, 1) a tabular dataset that contains the prediction and 2) an `OutputFileDatasetConfig` that can be used for the sequential pipeline steps. The `inference_script_tcn.py` performs a rolling evaluation on the test set with the step size of 24 hours. This generates the same results as if the model was put in production and generated a 24-hour forecast once a day using the most recent data context. This is an efficient way of testing historical model performance at a tiny fraction of the compute and time commitment costs. For a more detailed information on rolling evaluation see the this [document](https://learn.microsoft.com/en-us/azure/machine-learning/v1/how-to-auto-train-forecast-v1#evaluating-model-accuracy-with-a-rolling-forecast) for a more detailed explanation.\n",
+    "\n",
+    "<!-- The test_step uses the following arguments:<ul><li>`spearman_correlation`</li><li>`normalized_root_mean_squared_error`</li><li>`r2_score`</li><li>`normalized_mean_absolute_error`</li></ul> -->\n",
+    "\n",
+    "The `PythonScriptStep` uses the following arguments:<ul>\n",
+    "    <li> `model_name`: the name under which the model was registered in the workspace. </li>\n",
+    "    <li> `ouput_dataset_name`: name of the dataset under which the test set predictions will registered in the datastore</li>\n",
+    "    <li> `test_dataset_name`: name of the test dataset </li>\n",
+    "    <li> `target_column_name`: target column name</li>\n",
+    "    <li> `time_column_name`: time column name</li>\n",
+    "    <li> `output_path`: output data that is obtained from prediction step. </li>\n",
+    "    <li> `run_rolling_evaluation`: Boolean parameter. Set to True to run rolling evaluation. Otherwise, runs a regular forecast for the entire test set. </li>\n",
```

Comment:
Instead of "regular" maybe "recursive" - or briefly explain that this means the forecaster will extend predictions beyond the horizon by feeding predictions back into the model (which will compound forecasting error the farther out it goes)

Reply:
Good catch. Thank you. Modified the description.